src/Microsoft.AspNet.Mvc.ModelBinding/IModelMetadataProvider.cs

```diff
@@ -10,10 +10,10 @@ namespace Microsoft.AspNet.Mvc.ModelBinding
 {
     public interface IModelMetadataProvider
     {
-        IEnumerable<ModelMetadata> GetMetadataForProperties([NotNull] Type containerType);
-
         ModelMetadata GetMetadataForType([NotNull] Type modelType);
 
-        ModelMetadata GetMetadataForParameter([NotNull] MethodInfo methodInfo, [NotNull] string parameterName);
+        IEnumerable<ModelMetadata> GetMetadataForProperties([NotNull] Type modelType);
```

Comment:
I am wondering what led to the removal of GetMetadataForProperty and not GetMetadataForProperties?